lib/pymedphys/_vendor/pylinac/winstonlutz.py

```diff
@@ -58,76 +58,70 @@
 ALL = "All"
 
 
-def get_latest_wlimage():
-    VERSION_TO_CLASS_MAP = get_version_to_class_map()
-    return VERSION_TO_CLASS_MAP[pylinac.__version__]
+class WLImageCurrent:
```

Comment:
I have adjusted this class to utilise composition instead of inheritance so that I can define it at a top level of the file instead of scoping it under a function.

In the end, composition is preferred anyway. This class wrapper here is part of the review.